monitoring/healthz/src/DiscoveryHealth.tsx

```diff
@@ -76,6 +78,16 @@ function HealthRow({ isContent, sp }: { isContent: boolean; sp: SP }) {
     bytesToGb(health.database_size) || bytesToGb(health.databaseSize)
   const autoUpgradeEnabled =
     health.auto_upgrade_enabled || health.autoUpgradeEnabled
+  const getPeers = (str: string) => {
+    const match = str.match(/Peers: (.)/)
+    return (match && match[1]) ? match[1] : "no peers found"
+  }
+  const getProducing = (str: string) => {
+    return (!str.includes("The node stopped producing blocks.")).toString()
+  }
+  // currently discprov does not expose the address of its internal chain instance
+  const isSigner = (str: string) => getProducing(str)
```

Comment:
this is there we'd do that comparison I mentioned in the PR description, knowing if the node can produce blocks though is enough to infer if it can sign or not